File: TM 15 C#-Basics/C#-Basics/Arrays/program3.cs
// Tek Raj Joshi
// Superset ID: 1368453

using System;
class Program3 {
  
    public static void Main()
    {
        int[] arr = new int[] {1, 9, 6, 7, 5, 9};
        Array.Sort(arr);
        Array.Reverse(arr);
        foreach(int value in arr)
        {
            Console.Write(value + " ");
        }
    }
}

File: TM 6 Exception Handling/Exception Handling/program3.py
# Tek Raj Joshi
# Superset ID: 1368453

try:
    with open('filename.txt', mode='r', encoding = 'utf-8') as f:
        print(f.read())
        f.close()
except Exception as e:
    print("Oops! ", e, "occured.")
